src/backend/marsha/core/views.py

```diff
@@ -149,6 +150,7 @@ def _get_app_data(self):
                 pass
 
             app_data["jwt"] = str(jwt_token)
+            app_data["sentry_dsn"] = os.environ.get("DJANGO_SENTRY_DSN")
```

Comment:
This is confusing because there is already a SENTRY_DSN for the backend: https://github.com/openfun/marsha/blob/master/src/backend/marsha/settings.py#L276

Why not go through Django configuration for this sentry DSN as well and differentiate them explicity by their name? `BACKEND_SENTRY_DSN` and `FRONTEND_SENTRY_DSN` ? Unless they can be the same with a bakend / frontend variable passed in context?

Reply:
@lunika are you sure you are really using the same because the backend one is set with:
```
sentry_dsn = values.Value(None, environ_name="SENTRY_DSN")
```
so no DJANGO_ in front... ?

To answer your question, I think it's better to use the same DSN providing you add a parameter to differentiate both.

Reply:
Yes I'm sure I'm using the same